internal/services/catalog_read_service/internal/products/features/searching_products/v1/endpoints/search_products_endpoint.go

```diff
@@ -79,7 +79,9 @@ func (ep *searchProductsEndpoint) handler() echo.HandlerFunc {
 			ListQuery:  request.ListQuery,
 		}
 
-		if err := ep.Validator.StructCtx(ctx, query); err != nil {
+		// Am I right?  this one is a validator.StructCtx with   ctx, can I just make it to query.Validate?
+		// if err := ep.Validator.StructCtx(ctx, query); err != nil {
+		if err := query.Validate(); err != nil {
```

Comment:
It is fine, `ep.validator.StructCtx` was for checking validations tags by `go-playground/validator`. Now we could remove it